app/src/main/res/values/styles.xml

```diff
@@ -1,9 +1,11 @@
 <resources>
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
```

Comment:
I think Toolbar and NoActionBar is not neccessary - you can use normal ActionBar just like in the library. Make an exception for first Activity, that will have different theme (without actionbar or toolbar). This way we will have minimal changes.